src/webui/api/torrentscontroller.cpp

```diff
@@ -474,16 +474,31 @@ void TorrentsController::trackersAction()
 
     for (const BitTorrent::TrackerEntry &tracker : asConst(torrent->trackers()))
     {
+        int numPeers = -1;
+        int numSeeds = -1;
+        int numLeeches = -1;
+        int numDownloaded = -1;
+        for (const auto &endpoint : tracker.stats)
+        {
+            for (const auto &protocolStats : endpoint)
```

Comment:
Please use full type in both loops instead of `auto`.

The same in trackerlistwidget.cpp.